api/config/config_local.json

```diff
@@ -94,7 +94,7 @@
     "useKeylessDelegatedCredentials": true,
     "sendFreeTierAlertEmails": false,
     "enableMoodleV2Api": true,
-    "requireInstitutionalVerification": false,
+    "requireInstitutionalVerification": true,
```

Comment:
If we are going to set this to true on local, we should probably do it on test too, just to match.